app/templates/events/trackVolunteers.html

```diff
@@ -31,30 +32,35 @@ <h1 class="text-center">{{event.name}}</h1>
     </li>
   </ul>
 </div>
-<label class='d-none' id="eventLength">{{eventLength}}</label>
-<input class="form-control" id="eventID" name="event" value="{{event}}" type='hidden' />
-<div class="flex-container" style="padding: 5px;">
-  <div class='form-group d-inline-flex'>
-    <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add volunteer</a>
-    <a class="btn btn-warning" role="button" href= "/eventsList/{{event.id}}/kiosk" style="margin:5px;">Kiosk Entry</a>
+
+<h2 class ="text-center"> Volunteers </h2>
+<form action="#" method="post"><br>
+  <label class='d-none' id="eventLength">{{eventLength}}</label>
+  <input class="form-control" id="eventID" name="event" value="{{event}}" type='hidden' />
```

Comment:
Don't we want `event.id`? You're just outputting a whole object's string representation